.github/workflows/changelog.yml

```diff
@@ -1,9 +1,15 @@
 name: changelog
 
 on:
+  push:
```

Comment:
Does the push even still have the label data? That seems weird, just double checking!

Reply:
Hmm I did not understand that ? Do you mean we should put this under "pull_request" ? Not an expert in github actions :)

Reply:
I think so? (It's already there).

It feels weird to check the labels of a `push` event, I don't know if that even works..

Reply:
Make sense, thank you !